packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -43,6 +56,25 @@ export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void
   const abortController = new AbortController();
 
   logger.info("Lodestar", {network, version, commit});
+  if (poolSizeIsNanWarn) {
```

Comment:
Previously this was part of bls pool implementation which I liked more, any reasons this was moved here?

Reply:
Cayman requested that in his review we did last night.  To have a cli flag option (and env) to set as well as to have the logging near where the flag was set.  We talked about using console.log in apply presets but that wont show up in the written logs of the user so I wanted to move the logging to where the logger was available.